### PR TITLE
Addressing the non-support of Java 8

### PR DESCRIPTION
**Pull request by Serdaro**

### CODE REVIEW

**Comment by Serdaro** [comment]
Will apply the relevant changes to the main branch.

**Review by angelasimms — APPROVED** [review]
LGTM

File: docs/modules/deploy/pages/versioning-compatibility.adoc [hunk 1]
```diff
@@ -135,16 +135,13 @@ Hazelcast Platform has been tested against the following JVMs. It may run on oth
 |===
 |JDK | Platform {full-version}
 
-|Amazon Corretto 17
-|✓
-
-|Amazon Corretto 11
+|AdoptOpenJDK 11
```
Comment (frant-hartm) [inline docs/modules/deploy/pages/versioning-compatibility.adoc:138]: Where did this come from? We don't test with `AdoptOpenJDK 11`

File: docs/modules/deploy/pages/versioning-compatibility.adoc [hunk 2]
```diff
@@ -171,15 +162,14 @@ Hazelcast Platform has been tested against the following JVMs. It may run on oth
 |Red Hat build of OpenJDK 11
 |✓
 
-|Microsoft Build of OpenJDK 11
-|✓
-
```
Comment (frant-hartm) [inline docs/modules/deploy/pages/versioning-compatibility.adoc:176]: Why is this gone? See the complete list here:
https://docs.google.com/document/d/1BzmPiXuC9MyJ4WDwjvt1f0dJKzWoudT_rqT0TKUPGt0/edit#heading=h.vhojv0k3bqco

**Review by frant-hartm — CHANGES_REQUESTED** [review]

File: docs/modules/deploy/pages/versioning-compatibility.adoc (hunk 2 above)
Comment (Serdaro) [reply docs/modules/deploy/pages/versioning-compatibility.adoc:176]: Same as above.

**Comment by Serdaro** [comment]
Updated as requested.

**Comment by Serdaro** [comment]
@frant-hartm , Azul Zing versions should be included too?

**Comment by frant-hartm** [comment]
@Serdaro yes, and also IBM Semeru.

**Comment by Serdaro** [comment]
Done

File: docs/modules/deploy/pages/versioning-compatibility.adoc (hunk 1 above)
Comment (Serdaro) [reply docs/modules/deploy/pages/versioning-compatibility.adoc]: Copy paste error (from an old google docs)

**Pull request merged**